app/src/organisms/Devices/ProtocolRun/SetupLiquids/LiquidDetailCard.tsx

```diff
@@ -42,18 +47,25 @@ export function LiquidDetailCard(props: LiquidDetailCardProps): JSX.Element {
     labwareWellOrdering,
   } = props
   const trackEvent = useTrackEvent()
+  const isOdd = useSelector(getIsOnDevice)
```

Comment:
Brent created this function and he always use `isOnDevice` so I recommend you to use the same name.

```suggestion
  const isOnDevice = useSelector(getIsOnDevice)
```

Reply:
yup, my bad. i think i was lazy when imported it and used `isOdd` 😅